### PR TITLE
Add test for AzureRmSqlServerActiveDirectoryAdministrator API

### PR DESCRIPTION
**Pull request by jaczhan**
This is adding test for new Swagger AzureRmSqlServerActiveDirectoryAdministrator API for ActiveDirectoryOperations include:
1. Create/Update ActiveDirectory Admin
2. Get ActiveDirectory Admin
3. Delete ActiveDirectory Admin
4. Get list of ActiveDirectory Admin

Swagger PR at: https://github.com/Azure/azure-rest-api-specs/pull/1203

### CODE REVIEW

File: src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                ServerAzureADAdministrator createResult = sqlClient.ServerAzureADAdministrators.CreateOrUpdateAzureADAdministrator(resourceGroup.Name, server.Name, newAdmin);
+                
+                Assert.Equal(aadAdmin, createResult.Login);
+
+                // Get the current Active Directory Admin
+                ServerAzureADAdministrator getResult = sqlClient.ServerAzureADAdministrators.Get(resourceGroup.Name, server.Name);
+                Assert.Equal(aadAdmin, getResult.Login);
+                Assert.Equal(new Guid("5e90ef3b-9b42-4777-819b-25c36961ea4d"), getResult.Sid);
+                Assert.Equal(new Guid("72f988bf-86f1-41af-91ab-2d7cd011db47"), getResult.TenantId);
+
```
Comment (nathannfan) [inline src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs]: Add a check for list here too?

**Review by nathannfan — APPROVED** [review]
Looks good to me if it passes validation.

File: src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj [hunk 2]
```diff
@@ -23,4 +23,8 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.5" />
```
Comment (shahabhijeet) [inline src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj]: @jaczhan you don't need to add these references, they get implicitly added.

File: src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Sql.Tests
+{
+    public class AzureRmSqlServerActiveDirectoryAdministratorTest
+    {
+        [Fact]
```
Comment (shahabhijeet) [inline src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs]: @jaczhan please record test using latest nugets (e.g. 1.5.0 version of Resource manager) Currently you are using 1.1.0-preview version of Resource Manager.
All new tests should be using latest version of Resource Manager.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs (hunk 3 above)
Comment (jaczhan) [inline src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs]: how do I switch to the latest nugets?

Comment (jaczhan) [inline src/SDKs/SqlManagement/Sql.Tests/AzureRmSqlServerActiveDirectoryAdministratorTest.cs]: just updated

File: src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj [hunk 4]
```diff
@@ -10,4 +10,7 @@
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
   </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />
```
Comment (shahabhijeet) [inline src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj]: @jaczhan any reason you need this nuget in your SQL package?
The requirement was only for recording new test, so your other change is all you need.
You don't want additional dependency on your SQL SDK package.
Please revert "this" particular change.
Please keep your other change that you made in the test project (that is what I had asked for)

File: src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj [hunk 5]
```diff
@@ -23,4 +26,7 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.6.0" />
```
Comment (shahabhijeet) [inline src/SDKs/SqlManagement/Sql.Tests/Sql.Tests.csproj]: @jaczhan you do not need this, this is being added to every project.
Any reason you had to add reference to TestFramework?

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by jaczhan — COMMENTED** [review]
remove Resource Manager 1.6

**Comment by shahabhijeet** [comment]
@jaczhan add to whitelist

**Comment by jaczhan** [comment]
Adding test for Swagger AzureRmSqlServerActiveDirectoryAdministrator

https://github.com/Azure/azure-rest-api-specs/pull/1203

**Review by jaczhan — COMMENTED** [review]
changes made to fix 1.6 tag

**Comment by jaredmoo** [comment]
We will rerecord the tests separately - https://github.com/Azure/azure-sdk-for-net/issues/3267

**Comment by shahabhijeet** [comment]
@jaczhan looks good, as soon as CI passes this can be merged.

**Pull request merged**